go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/openshift/client-go v0.0.0-20210521082421-73d9475a9142
 	github.com/sigstore/cosign/v2 v2.0.0
 	github.com/spf13/viper v1.15.0
+	github.com/stretchr/testify v1.8.1
```

Comment:
We are using `gotest.tools` for assertions and stuff (`gotest.tools/v3 v3.4.0`), so if we could stick to it that would be nice(r) 👼🏼

Reply:
@vdemeester I think I can switch to `gotest.tools` for assertions easily.

Shall I continue with `github.com/stretchr/testify` for `suite interface`? 
do we have a similar concept in `gotest.tools`?

Reply:
@jkandasa we don't, but as discussed offline, it's fine to use testify, let's not block on this.